.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature request
+about: Marked doesn't do this thing and I think it should
+
+---
+

```

Comment:
Appreciating the updates to the feature request. Gonna be catching up on some things elsewhere but want to drop by as I'm going through emails.